test_environments/message_and_event_brokers/kafka/no_authentication/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+---
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:7.3.2
+    container_name: zookeeper
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-kafka:7.3.2
+    container_name: broker
```

Comment:
It is rarely that anyone would need to spin every Kafka setup we have here.
Also it would be very time consuming to provide setup with one Zookeeper and multiple Kafka brokers with different authentication setup.
I have updated names of every container which will be run.
I have updated the ports on which each setup is exposed and on which it runs in internal network. Each setup creates and uses it's own network (default feature of docker-compose) which is good for separation of concerns .